wire-tests/src/test/java/com/squareup/wire/java/KotlinGeneratorTest.kt

```diff
@@ -88,7 +88,10 @@ class KotlinGeneratorTest {
         .addImport("com.squareup.wire.kotlin", "decodeMessage")
         .build()
     val code = fileSpec.toString()
-    assertTrue(code.contains("object CREATOR : Parcelable.Creator<PhoneNumber>"))
+    assertTrue(code.contains("companion object"))
+    assertTrue(code.contains("@JvmStatic"))
+    assertTrue(code.contains("val CREATOR"))
+    assertTrue(code.contains("object : Parcelable.Creator<PhoneNumber>"))
```

Comment:
I'm skeptical of this testing strategy since the integration test already covers all of this implicitly

Reply:
Agree, and `assertTrue` is particularly unhelpful. However, I don't think integration testing is configured properly, since my changes didn't break any tests except for this one.